spec.html

```diff
@@ -20390,7 +20390,7 @@ <h1>Runtime Semantics: InstantiateFunctionObject</h1>
       </emu-grammar>
       <emu-alg>
         1. Let _name_ be StringValue of |BindingIdentifier|.
-        1. Let _F_ be ! OrdinaryFunctionCreate(%AsyncGenerator%, |FormalParameters|, |AsyncGeneratorBody|, ~non-lexical-this~, _scope_).
+        1. Let _F_ be ! OrdinaryFunctionCreate(%AsyncGeneratorFunction.prototype%, |FormalParameters|, |AsyncGeneratorBody|, ~non-lexical-this~, _scope_).
```

Comment:
i see this as an equivalent and editorial choice, unless we're trying to get rid of `%AsyncGenerator%` entirely - which we could do if no other specs reference it.